add march 2025 meetup (#13)
julia-users-helsinki/julia-users-helsinki.github.io: page/index.md

**If you would like to stay up-to-date with upcoming Julia related events in Helsinki area, join the Julia Users Helsinki [mailing list](https://groups.google.com/g/julia-users-helsinki).**

> Check out our next get together on **March 26th 2025**!

Reviewer: ```suggestion
Check out our next get together on **April 9th 2025**!
```

\begin{section}{title="Upcoming Events"}

> ### Julia at Murata

Author: @jaakkor2 do you have a better name in mind?

> ### Julia at Murata

Reviewer: ```suggestion
### Discover Innovation at Murata
```

> Learn how Murata uses Julia in their work! **Register by TBA**

Author: maybe a nicer description?

Reviewer 2: @jaakkor2, Maybe add a one-liner explaining what is Murata core business; I suspect some won't know the company. I would also consider the possibility of attending remotely (including for myself 😄)

Author: fwiw, we discussed with Jaakko to move this to April 9.4 to have more time to advertise

> Learn how Murata uses Julia in their work! **Register by TBA**

Reviewer: ```suggestion
[Murata](http://muratafinland.com/en/) designs, develops, and manufactures MEMS accelerometers, inclinometers, and gyro sensors in Vantaa, Finland.  We warmly invite you to join us for an exclusive on-site meetup where you will

* learn how Murata uses Julia language in development of inertial sensors,
* get a glimpse into the production on a window tour,
* experience a product demonstration.

Registration is required. Stay tuned for more information on how to register.
```

Author: I was thinking to collect registrations via google form (and indeed this PR has a fresh link) was that a problem?

Author: if it's ok, we can remove the "stay tuned" part

Reviewer: I was confused by `**Register by TBA**`.

Author: Ah right, when do you need to communicate the list of participants to murata at latest?

Reviewer: One day before the event.  Arrival instruction will be mailed in pdf format.

- **where**: Myllynkivenkuja 6, 01620 Vantaa

> - **when**: Wednesday 26th March, 15:30 -- 17:30

Reviewer: ```suggestion
- **when**: Wednesday 9th April, 15:45 -- 18:00
```

- **Speakers**: Jaakko Ruohio

\begin{center}
~~~
<a class="btn btn-primary btn-lg" href="https://forms.gle/cLsizZBZhi7qnZAB9" target="_blank" rel="noreferrer noopener" role="button">Register</a>
~~~

\end{center}

\end{section}

\begin{section}{title="Past Events"}
\begin{center}
\begin{table}{class="table-stripped"}
|**When**|**Topic**|**Speaker**|**Video**|
|:------:|:-------:|:---------:|:-------:|

> | December 2025 | Introduciton to Julia | Luca Ferranti and Fabricio Oliveira ||

Reviewer: ```suggestion
| December 2025 | Introduction to Julia | Luca Ferranti and Fabricio Oliveira ||
```

\begin{section}{title="Upcoming Events"}

### Discover Innovation at Murata

[Murata](http://muratafinland.com/en/) designs, develops, and manufactures MEMS accelerometers, inclinometers, and gyro sensors in Vantaa, Finland.  We warmly invite you to join us for an exclusive on-site meetup where you will

* learn how Murata uses Julia language in development of inertial sensors,
* get a glimpse into the production on a window tour,
* experience a product demonstration.

> Registration is required. Stay tuned for more information on how to register.

Author: ```suggestion
Registration is required. **Please register by April 7th**
```

- **where**: Myllynkivenkuja 6, 01620 Vantaa
- **when**: Wednesday 9th April, 15:45 -- 18:00
- **Speakers**: Jaakko Ruohio

\begin{center}
~~~
<a class="btn btn-primary btn-lg" href="https://forms.gle/cLsizZBZhi7qnZAB9" target="_blank" rel="noreferrer noopener" role="button">Register</a>
~~~

\end{center}

\end{section}

\begin{section}{title="Past Events"}
\begin{center}
\begin{table}{class="table-stripped"}
|**When**|**Topic**|**Speaker**|**Video**|
|:------:|:-------:|:---------:|:-------:|

> | December 2025 | Introduction to Julia | Luca Ferranti and Fabricio Oliveira ||

Reviewer: ```suggestion
| December 2024 | Introduction to Julia | Luca Ferranti and Fabricio Oliveira ||
```
